test/spec.index.js

```diff
@@ -42,12 +57,79 @@ describe('Template Mixins', function () {
             Hogan.compile.restore();
         });
 
-        describe('input-text', function () {
+        describe('with two arguments', function () {
 
-            beforeEach(function () {
-                middleware = mixins({}, { translate: translate });
+            // template mixins now only accepts one argument
+            it('shows a deprecation warning', function () {
+                proxiedMixins({}, {});
+                deprecate.should.have.been.calledOnce;
+            });
+
+            it('merges static field options with other options', function () {
+                middleware = proxiedMixins({
+                    'field-name': {
+                        validate: 'required'
+                    }
+                }, {
+                    sharedTranslationsKey: 'custom'
+                });
+                middleware(req, res, next);
+                res.locals['input-text']().call(res.locals, 'field-name');
+                render.should.have.been.called;
+                render.should.have.been.calledWithExactly(sinon.match({
+                    required: true,
+                    label: 'custom.fields.field-name.label'
+                }));
+            });
+
+        });
+
+        describe('with one argument', function () {
+
+            it('doesn\'t warn when whitelisted options are passed', function () {
+                proxiedMixins({ sharedTranslationsKey: 'custom', fields: {} });
+                deprecate.should.not.have.been.called;
+            });
+
+            it('warns when options don\'t contain any standard properties', function () {
+                proxiedMixins({ 'field-name': {} });
+                deprecate.should.have.been.called;
+            });
+
+            it('adds static field config as a field option', function () {
+                middleware = proxiedMixins({
+                    'field-name': {
+                        validate: 'required'
+                    }
+                });
+                middleware(req, res, next);
+                res.locals['input-text']().call(res.locals, 'field-name');
+                render.should.have.been.called;
+                render.should.have.been.calledWithExactly(sinon.match({
+                    required: true
+                }));
             });
 
+        });
+
+    });
+
+    describe('with stubbed Hogan', function () {
```

Comment:
This is showing up as new but it's existing code. We're stubbing the render method of Hogan to check the properties exposed to the view. I'll leave it as is for now.